tests/nemo_text_processing/hy/data_inverse_text_normalization/test_cases_money.txt

```diff
@@ -0,0 +1,10 @@
+մեկ դոլար~1 $
+մեկ ռուբլի~1 ₽
+երեսունմեկ ռուբլի~31 ₽
+տասնվեց հազար ամն դոլար~16000 $
```

Comment:
տասնվեց հազար ամն դոլար~16000 $ =(it is translated as) sixteen thousand USA dollars ~ 16000 $
տասնութ հազար դոլար~18000 $ = (it is translated as) eighteen thousand dollars ~ 18000 $
There is a difference between the line 4 and 5 format ամն=USA

Reply:
In Inverse Text Normalization there is no difference whether it is "sixteen thousand us dollars" or "sixteen thousand dollars" both are transformed into "$16000". I did the same for Armenian so "ամն դոլար" or "դոլար" both are transformed into "$".